docs/docusaurus.config.js

```diff
@@ -123,8 +121,7 @@ const config = {
 						position: 'left',
 					},
 					{
-						to: 'docs/api/', // 'api' is the 'out' directory
-						activeBasePath: 'docs',
+						to: 'api', // 'api' is the 'out' directory
 						label: 'API',
 						position: 'left',
 					},
```

Comment:
Its not part of this PR but for now ( until we have main 4.0.0 released and as default branch on repo ) lets make:
`href: 'https://github.com/ChainSafe/web3.js',`
to
`href: 'https://github.com/ChainSafe/web3.js/tree/4.x',`